receiver/awsecscontainermetricsreceiver/README.md

```diff
@@ -142,15 +142,15 @@ aws.ecs.cluster.name | aws.ecs.cluster.name
 aws.ecs.task.family  | aws.ecs.task.family
 aws.ecs.task.arn     | aws.ecs.task.arn
 aws.ecs.task.id      | aws.ecs.task.id
-aws.ecs.task.version | aws.ecs.task.version
+aws.ecs.task.revision | aws.ecs.task.revision
 aws.ecs.service.name | aws.ecs.service.name
 cloud.availability_zone | cloud.availability_zone
 cloud.account.id | cloud.account.id
 cloud.region | cloud.region
 aws.ecs.task.pull_started_at | aws.ecs.container.started_at
 aws.ecs.task.pull_stopped_at | aws.ecs.container.finished_at
 aws.ecs.task.known_status | aws.ecs.container.know_status
-aws.ecs.task.launch_type | aws.ecs.task.launch_type
+aws.ecs.launch_type | aws.ecs.launch_type
```

Comment:
Will this have a backwards compatibility issue? Is the current naming convention being used downstream anywhere?

Reply:
This is not a breaking change: we still will be sending both attributes after this change, but the docs will reflect only the new name.